### PR TITLE
Bulk update integer display width

### PR DESCRIPTION
**Pull request by dveeden**
### What is changed, added or deleted? (Required)

Related to:
- https://github.com/pingcap/docs/pull/19159
- https://github.com/pingcap/tidb/pull/56529

What this does is basically the following, with some manual changes:

```shell
git grep -il '[^a-z]int([0-9]*)' | grep -vE '^(information-schema|releases)' | xargs sed -i 's/[^a-z]int([0-9]*)/ int/gi'
git grep -il '[^a-z]tinyint([0-9]*)' | grep -vE '^(information-schema|releases)' | xargs sed -i 's/[^a-z]tinyint([0-9]*)/ tinyint/gi'
git grep -il '[^a-z]smallint([0-9]*)' | grep -vE '^(information-schema|releases)' | xargs sed -i 's/[^a-z]smallint([0-9]*)/ smallint/gi'
git grep -il '[^a-z]mediumint([0-9]*)' | grep -vE '^(information-schema|releases)' | xargs sed -i 's/[^a-z]mediumint([0-9]*)/ mediumint/gi'
git grep -il '[^a-z]bigint([0-9]*)' | grep -vE '^(information-schema|releases)' | xargs sed -i 's/[^a-z]bigint([0-9]*)/ bigint/gi'
```

Things to consider:
- This should not affect `releases/`
- We should be careful to not break formatting of table based output 
- We should be careful to not break examples that rely on display width
- The display width may be visible in information_schema information. Also this has a lot of table based output that we should be careful not to break the formatting of.
- Display width can still be in (older)MySQL and MariaDB output (for migration info etc)
- `tinyint(1)` is a special case as it is an alias for `bool` (see `data-type-numeric.md`)
- non-sql should not be replaced (e.g. `decimal.NewFromInt(10000)`)
- This should not affect `tidb-cloud/`

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.5 (TiDB 8.5 versions)
- [ ] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dm/dm-table-routing.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ To extract the source information of upstream sharded tables to the merged table
 
 ```sql
 CREATE TABLE `test`.`t` (
-    a int(11) PRIMARY KEY,
+    a int PRIMARY KEY,
```
Comment (mjonss) [inline dm/dm-table-routing.md:119]: Similar here I assume MySQL is the upstream, but I have no issues with updating these as well (unless the output is still including the display width from MySQL or DM).

File: dm/dm-table-routing.md [hunk 2]
```diff
@@ -184,7 +184,7 @@ mysql> select * from test.t;
 ```sql
 CREATE TABLE `test`.`t` (
     c_table varchar(10) DEFAULT NULL,
-    a int(11) PRIMARY KEY,
+    a int PRIMARY KEY,
```
Comment (mjonss) [inline dm/dm-table-routing.md:187]: Same here, where does it origin from and will it be with or without display width?

File: dm/manually-handling-sharding-ddl-locks.md [hunk 3]
```diff
@@ -331,7 +331,7 @@ The operation processes are:
                 "mode": "pessimistic"
                 "owner": "mysql-replica-02",
                 "DDLs": [
-                    "USE `shard_db`; ALTER TABLE `shard_db`.`shard_table` ADD COLUMN `c2` int(11);"
+                    "USE `shard_db`; ALTER TABLE `shard_db`.`shard_table` ADD COLUMN `c2` int;"
```
Comment (mjonss) [inline dm/manually-handling-sharding-ddl-locks.md:334]: Is this generated by MySQL or DM? Will it use display width or not?

File: tidb-cloud/serverless-driver-kysely-example.md [hunk 4]
```diff
@@ -93,7 +93,7 @@ To complete this tutorial, you need the following:
 
    ```sql
    CREATE TABLE `test`.`person`  (
-     `id` int(11) NOT NULL AUTO_INCREMENT,
+     `id` int NOT NULL AUTO_INCREMENT,
```
Comment (mjonss) [inline tidb-cloud/serverless-driver-kysely-example.md:96]: For create this is OK, but I assume for `SHOW CREATE TABLE` etc. it should be reflected in the correct TiDB Serverless version documentation?

File: tidb-cloud/tidb-cloud-import-local-files.md [hunk 5]
```diff
@@ -90,7 +90,7 @@ To import only some specified columns, you can use the MySQL client to connect y
 
 ```sql
 CREATE TABLE `import_test` (
-  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `id` int NOT NULL AUTO_INCREMENT,
```
Comment (mjonss) [inline tidb-cloud/tidb-cloud-import-local-files.md:93]: `CREATE TABLE` is OK, but for output I assume the documentation will be synced with different repo/version?

Comment (dveeden) [reply tidb-cloud/tidb-cloud-import-local-files.md:93]: @qiancai while this `CREATE TABLE` statement would work on both current and future versions of TiDB Cloud Serverless, maybe we should exclude `tidb-cloud/` for now?

@mjonss cloud uses a different TOC: `TOC-tidb-cloud.md` but the same repo. Not sure what branch is used now

File: dm/dm-manage-schema.md [hunk 6]
```diff
@@ -124,7 +124,7 @@ binlog-schema list -s mysql-replica-01 task_single db_single t1
     "sources": [
         {
             "result": true,
-            "msg": "CREATE TABLE `t1` ( `c1` int(11) NOT NULL, `c2` int(11) DEFAULT NULL, PRIMARY KEY (`c1`)) ENGINE=InnoDB DEFAULT CHARSET=latin1 COLLATE=latin1_bin",
+            "msg": "CREATE TABLE `t1` ( `c1` int NOT NULL, `c2` int DEFAULT NULL, PRIMARY KEY (`c1`)) ENGINE=InnoDB DEFAULT CHARSET=latin1 COLLATE=latin1_bin",
```
[the remark this answers is not in the document]
Comment (dveeden) [reply dm/dm-manage-schema.md:127]: @lance6716 Should we exclude `dm/` for now or not?

[the remark this answers is not in the document]
Comment (lance6716) [reply dm/dm-manage-schema.md:127]: dm will only save TableInfo and recover the SHOW CREATE TABLE statement using TiDB functions, so it should also be updated

File: tidb-cloud/tidb-cloud-import-local-files.md (hunk 5 above)
Comment (qiancai) [reply tidb-cloud/tidb-cloud-import-local-files.md:93]: @dveeden Yes, I agree with you. As the latest TiDB Cloud specific content is still maintained in [release-7.5 -> tidb-cloud](https://github.com/pingcap/docs/tree/release-7.5/tidb-cloud) folder, we can skip this change in the master -> tidb-cloud folder.  When v8.5 becomes the default version of TiDB Dedicated, we can update the content in the tidb-cloud folder accordingly.

File: basic-sql-operations.md [hunk 7]
```diff
@@ -100,7 +100,7 @@ For example, to create a table named `person` which includes fields such as numb
 
 ```sql
 CREATE TABLE person (
-    id INT(11),
+    id int,
```
Comment (Oreoxmt) [inline basic-sql-operations.md:103]:
```suggestion
    id INT,
```

Ensure consistent capitalization of SQL data types (`INT`, `VARCHAR `, `DATE`) in this SQL statement.

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: basic-sql-operations.md (hunk 7 above)
Comment (Oreoxmt) [reply basic-sql-operations.md:103]: @dveeden PTAL

**Review by qiancai — APPROVED** [review]

**Pull request merged**